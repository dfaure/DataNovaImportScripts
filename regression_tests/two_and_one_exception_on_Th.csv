17781A;APATOU BP;2020-12-22;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-11;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-12;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-14;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-13;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-15;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-14;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-16;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-07;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-05;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-23;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-21;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-17;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-28;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-09;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-15;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-06;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-29;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-08;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-04;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-18;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-30;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-24;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-19;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2020-12-31;09:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-16;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-02;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-01-21;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-19;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-18;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-02;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-11;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-10;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-09;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-26;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-04;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-05;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-05;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-01;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-03;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-01;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-04;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-02;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-08;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-03-06;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-20;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-19;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-25;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-29;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-03;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-27;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-17;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-18;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-26;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-24;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-23;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-22;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-15;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-16;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-06;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-20;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-27;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-13;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-25;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-02-08;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-28;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-02-12;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-30;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-23;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-22;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-03-09;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-11;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-03-12;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-15;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-16;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
17781A;APATOU BP;2021-03-10;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-13;08:00-12:00;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-18;09:00-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
17781A;APATOU BP;2021-03-17;07:15-13:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
