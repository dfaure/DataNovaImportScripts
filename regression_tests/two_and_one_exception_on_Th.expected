17781A|APATOU BP|Mo-We,Fr 07:15-13:30; Th 09:00-13:30; Sa 08:00-12:00; PH off; 2020 Dec 24,2020 Dec 31 09:00-12:00; 2020 Dec 19,2020 Dec 26,2021 Feb 15-17,2021 Mar 11 off
